edenai_apis/utils/file_handling.py

```
@@ -0,0 +1,54 @@
+import base64
```

Pylint (4.0.2)

[error] Unrecognized option found: suggestion-mode

(E0015)